package.json

```diff
@@ -10,12 +10,13 @@
   "exports": {
     ".": {
       "require": "./dist/locally-unique-id-generator.js",
-      "import": "./dist/locally-unique-id-generator.esm.js"
+      "import": "./dist/locally-unique-id-generator.esm.js",
+      "svelte": "./dist/locally-unique-id-generator.esm.js"
     },
     "./package.json": "./package.json"
   },
   "scripts": {
-    "build": "rimraf dist && rollup -c rollup.config.js && tsc && mv src/*.d.ts dist && rm src/*.js*",
+    "build": "rimraf dist && rollup -c rollup.config.js && tsc && mv src/*.d.ts dist && rimraf -g src/*.js* && rimraf dist/src",
```

Comment:
A `.d.ts` was moved to `dist/src/xxx.d.ts`, probably a side-effect of upgraded packages… So now I remove it after each build.